### DISCUSSION TITLE
Welcome to orbit Discussions!

### DISCUSSION THREAD

**Discussion by edwinnglabs** [uber/orbit#355] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by melody188** [comment]
Hi I'm trying to use the orbit package to make some time series prediction. Since I need to make lots of predictions at the same time, I want to use parallel computing to expedite the procedure. So I used the Parallel function from joblib package in python. But it seems that there are some parallel procedure inside the package which contradict with my parallel function. Thus, it is not running stable. Do you have any recommended way to make parallelization using orbit? By the way, I also tried to add parameters cores = xx in DLTMAP, but it doesn't seem to make any difference. The time to take for running this function is the same no matter I choose cores = 1 or cores = 60. Did I use it correctly? Could any one give any any clues?

Example: here is the code where I specified the cores.
`
    dlt = DLTMAP(
        response_col=response_col,
        date_col=date_col,
        seasonality=48,
        prediction_percentiles=[0, 0.25, 0.5, 1, 99, 99.5, 99.75, 100],
        global_trend_option='flat',
                    cores = 5 
    )` 

I also tried something like this below:
'res = Parallel(n_jobs=60)(delayed(wrapper_function_to_make _one_time_prediction_using_orbit)(cid, watchtower_data, split_time, 
                          response_col, date_col, pred_len, trend_day) for cid in tqdm(non_empty_cid[(10*i):(10*(i+1))]))'
It seems that orbit also calls some parallel procedure. So when I call the parallel procedure from wrapper_function_to_make _one_time_prediction_using_orbit, it somehow has conflicts will sometimes induce errors and kill the codes unexpectedly
